modules/data.land/R/fia2ED.R

```diff
@@ -144,12 +144,13 @@ fia.to.psscss <- function(settings,
   query <- paste('SELECT p.cycle, p.statecd, p.measyear as time, p.cn as patch, ', 
                  'MIN(2-c.stdorgcd) as trk, AVG(c.stdage) as age, p.lat, p.lon, p.prev_plt_cn ',
                  'FROM plot as p LEFT JOIN cond as c on p.cn=c.plt_cn ',
-                 'WHERE p.lon >= ', lonmin, ' AND p.lon < ', lonmax,
-                 ' AND p.lat >= ', latmin, ' AND p.lat < ', latmax, 
-                 ' AND p.measyear >= ', min.year, ' AND p.measyear < ', max.year, 
+                 'WHERE p.lon >= ', lonmin, ' AND p.lon <= ', lonmax,
+                 ' AND p.lat >= ', latmin, ' AND p.lat <= ', latmax, 
+                 ' AND p.measyear >= ', min.year, ' AND p.measyear <= ', max.year, 
                  ' GROUP BY p.cn')
 
   pss <- db.query(query, con=fia.con)
+  if(nrow(pss) == 0) stop(logger.error("No pss data found."))
```

Comment:
Ah, OK. That's helpful and will work here. It might be helpful to have logger.warn throw a real warning message too. If nothing else the fact that it doesn't prevented me from testing for a warning using testthat::expect_warning(). I had edited logger.message to silently return the message so developers could use it however they want while still relying on logger.*. Just some thoughts. I'll change it here.